docs/en/observability/apm/feature-roles.asciidoc

```diff
@@ -21,60 +42,82 @@ For secure communication between APM Server and APM Agents, see <<apm-secure-age
-When possible, assign users the built-in roles to minimize the affect of future changes on your security strategy.
-If no built-in role is available, you can assign users the privileges needed to accomplish a specific task.
-In general, there are three types of privileges you'll work with:
+In general, there are three types of privileges you'll work with when creating roles:
 
 * **{es} cluster privileges**: Manage the actions a user can perform against your cluster.
 * **{es} index privileges**: Control access to the data in specific indices your cluster.
 * **{kib} space privileges**: Grant users write or read access to features and apps within {kib}.
+====
+
+Below are some common roles that APM Server users might need:
+
+* <<apm-privileges-to-publish-events,*Writer role*>>:
+Allows a user to publish events collected by APM Server, which is *required* to write to {es}.
+* <<apm-privileges-agent-central-config,*Central configuration management role*>>:
+Allows a user to view APM Agent central configurations, which is *required* when
+<<apm-agent-configuration,central configuration management>> is enabled (it is enabled by default).
+* <<apm-privileges-to-publish-monitoring,*Monitoring role*>>: Allows a user to publish monitoring data,
+view monitoring data, or both.
+* <<apm-privileges-rum-source-mapping,*RUM source mapping role*>>: Allows a user to read RUM source maps.
+
+// TO DO: Replace <do something> with a common task made possible by the listed roles
+.Example: Assigning multiple roles to an APM Server user
+*****
+If you use agent central configuration to manage APM agents, you use RUM source mapping,
+and you want to create an APM Server user who can <do something>,
+you would need to assign these three roles to the user:
+
+* <<apm-privileges-to-publish-events,Writer role>>
+* <<apm-privileges-agent-central-config,Central configuration management role>>
+* <<apm-privileges-to-publish-events,RUM source mapping role>>
+*****
 
 ////
 ***********************************  ***********************************
 ***********************************  ***********************************
 ////
 
+[float]
 [[apm-privileges-to-publish-events]]
-=== Grant privileges and roles needed for writing events
+=== Create a _writer_ role
 
-++++
-<titleabbrev>Create a _writer_ user</titleabbrev>
-++++
 
-APM users that publish events to {es} need privileges to write to APM data streams.
+APM users that publish events to {es} _must_ have privileges to write to APM data streams.
 
-[float]
-==== General writer role
+// Not sure if I captured this accurately...
+[NOTE]
+====
+This is not needed when APM Server doesn't write to {es} directly.
+For example, in some cases you may configure APM Server to write to another
+output like Logstash, Kafka, or any another output supported by libbeat.
+In these cases, another configured {es} role will need to be passed to
+<<apm-agent-config-elasticsearch,`apm-server.agent.config.elasticsearch`>>.
+====
```

Comment:
From @endorama:

>Documentation for `output.elasticsearch`, which `apm-server.agent.config.elasticsearch` link to, does not mention roles at all, so I think this ends up being too confusing.

My question:

>@endorama @carsonip can you be more specific about what should be changed about the note as it exists in the [latest commit]?

Reply:
I would update it as follow:
```suggestion
// Not sure if I captured this accurately...
[NOTE]
====
This is not needed when APM Server doesn't write to {es} directly.
For example, in some cases you may configure APM Server to write to another
output like Logstash, Kafka, or any another output supported by libbeat.
In these cases, different authentication credentials will need to be passed to
<<apm-agent-config-elasticsearch,`apm-server.agent.config.elasticsearch`>>.
====
```

The idea is that you don't pass a role to that configuration but either a username/password pair, an API key or TLS certificates. Removing the mention to roles I think removes the confusion.